api/charts/on-the-fly.js

```diff
@@ -61,12 +62,37 @@ module.exports = ({ layer, type, depth, areaInterest, scenario }) => {
     geometry: areaInterest,
   };
 
+  logger.debug(`Loading OTF results with request POST ${url} ${JSON.stringify(body)}`);
+
   return axios
     .post(url, body, {
       headers: { Accept: 'application/json' },
     })
-    .then(({ data: { data: { counts, bins, mean_diff, mean_years, mean_values, area_ha } } }) => ({
-      timeseries: parseTimeseriesData(mean_years, mean_values),
-      change: parseChangeData(counts, bins, mean_diff, area_ha),
-    }));
+    .catch(error => {
+      logger.warn(`Error loading OTF results: ${error.toString()}`);
+    })
```

Comment:
In my latest commit to `develop`, I've moved this below the `then` at line 98. If you catch errors here, JS will continue with the `then` below and crash because `response` will have no value.

Reply:
fixed